convert-to-json.py

```diff
@@ -15,22 +15,23 @@
 
 system_name = os.path.basename(os.path.realpath(folder_name))
 
-with open(os.path.join(folder_name, "scf.in")) as fhandle:
+with open(os.path.join(folder_name, "GaAs.scf.in")) as fhandle:
     scf_input = fhandle.read()
-with open(os.path.join(folder_name, "scf.out")) as fhandle:
+with open(os.path.join(folder_name, "GaAs.scf.out")) as fhandle:
     scf_output = fhandle.read()
-with open(os.path.join(folder_name, "matdyn.modes")) as fhandle:
+with open(os.path.join(folder_name, "GaAs.modes")) as fhandle:
     matdyn_modes = fhandle.read()
 
-pretty_name_dict = {"BaTiO_3": "BaTiO<sub>3</sub>"}
+pretty_name_dict = {"GaAs": "GaAs"}
 
-highsym_qpts_default = [[0, "Γ"], [20, "M"], [40, "K"], [60, "Γ"]]
+highsym_qpts_default = [[0, "L"], [40, "Γ"], [80, "K"], [100, "X"], [140, "Γ"]]
 highsym_qpts_dict = {
-    "BaTiO_3": [[0, "X"], [20, "Γ"], [40, "M"], [60, "Γ"], [100, "R"]],
+    "GaAs": [[0, "L"], [40, "Γ"], [80, "K"], [100, "X"], [140, "Γ"]],
```

Comment:
Here you should put all the qpts that do not use the default qpts defined above. I think this is all the 3d examples that you added. BaTiO3 should also be kept here.